autoload/neomake/makers/ft/python.vim

```diff
@@ -83,6 +83,11 @@ function! neomake#makers#ft#python#Flake8EntryProcess(entry) abort
     else
         let type = ''
     endif
+    let l:token = matchstr(a:entry.text, "'.*'")
+    if strlen(l:token)
+        let a:entry.length = strlen(l:token) - 2 " subtract the quotes
+    endif
```

Comment:
Makes sense, thanks for explaining!
@JelteF 
I have a local branch/stash to improve Python makers btw.. all my recent improvements are ways to get there.. the current one being to fix "serialize": I would like to run the "python" maker serialized, and when it bails out (syntax errors) skip the others.